src/common/transformations/src/transformations/common_optimizations/rms_fusion.cpp

```diff
@@ -18,7 +18,7 @@
 #include "transformations/utils/utils.hpp"
 
 namespace ov {
-namespace intel_gpu {
+namespace pass {
 
 static std::function<bool(ov::Output<ov::Node>)> constant_value(const float target_value) {
```

Comment:
@jane-intel Don't we have such predicate somewhere? If no, I think we can generalize it and put to some common header. Something like that:

```cpp
template <typename T>
ov::pass::pattern::op::ValuePredicate constant_value(const T target_value, const T threshold = std::numeric_limits<T>::epsilon()) {
    return pass::pattern::op::as_value_predicate([=](const ov::Output<ov::Node>& output) -> bool {
        return ov::op::util::has_constant_value<T>(output.get_node_shared_ptr(), target_value, threshold);
    });
}
```

Reply:
same predicate is in prelu_fusion.cpp